src/actions/index.js

```diff
@@ -1,26 +1,42 @@
+import {ProductActions} from "../common/ProductActions";
+import {getAllProducts} from "../services/productsAPI";
+
 export const addProductAction = product => ({
-    type: 'ADD_PRODUCT',
+    type: ProductActions.ADD_PRODUCT,
     ...product
 });
 
+export const getProductsAction = () => {
+    return (dispatch) => {
+        getAllProducts()
+            .then(response => {
+                dispatch(onDataRequestSuccessAction(response.data))
+            })
+            .catch(error => {
+                dispatch(onDataRequestErrorAction)
```

Comment:
here you need to invoke onDataRequestErrorAction which will return description for of redux-action